README.md

```diff
@@ -93,11 +93,27 @@ mkdir -p /opt/epics-tools/services/nasa/kafka
 cd /opt/epics-tools/services/nasa/kafka
 tar -xvf /tmp/kafka_2.13-3.9.0.tgz
 ```
+**Note:** It is recommended to keep a backup of the Kafka binaries (kafka_2.13-3.9.0.tgz) in a secure location. Older versions may be removed from the official site and become unavailable for future downloads.
 
 #### Optional Configuration
```

Comment:
@shroffk Is persisting the Kafka database actually optional for a deployment following these instructions?